src/main/java/net/juniper/netconf/Device.java

```diff
@@ -246,12 +246,12 @@ private Session loginWithPrivateKey(int timeoutMilliSeconds) throws NetconfExcep
         try {
             Session session = sshClient.getSession(userName, hostName, port);
             session.setConfig("userauth", "publickey");
-            session.setConfig("StrictHostKeyChecking", "no");
+            session.setConfig("StrictHostKeyChecking", isStrictHostKeyChecking() ? "yes" : "no");
```

Comment:
it seems to me you forgot to add `isStrictHostKeyChecking()` method

Reply:
That method is auto-generated by lombok. It will compile

Reply:
oh, I see. I missed that, sorry. What about IT?